pinc/charsuite-symbols-collection.inc

```diff
@@ -0,0 +1,53 @@
+<?php
+include_once($relPath."CharSuites.inc");
+
+$charsuite = new CharSuite("symbols-collection", _("Symbols collection"));
+$charsuite->description = _("Astronomical, zodiac and a limited number of apothecary and music symbols. The zodiac symbols are provided with the variation selector, VS15, to try to force them to display as ordinary charcters instead of as emoji. This may not work for all fonts.");
```

Comment:
should we link to some wiki about variation selectors?

Reply:
Yes - Andrew, Sharon and I chatted about this and one is forthcoming.

Reply:
The wiki page has been started, but not completed as of this moment. However, I went ahead and added a link to it -- it will hopefully be ready by the time we need it.